references/starter-templates/databases/postgres.md

```diff
@@ -15,16 +15,23 @@ build:
         repository: http://apt.postgresql.org/pub/repos/apt
         distribution: jammy-pgdg
         components: ["main"]
-
-launch:
-  steps:
     - type: command
       command: |
-        systemctl start postgresql.service
         echo 'postgres     ALL=NOPASSWD: ALL' | sudo tee /etc/sudoers.d/100-postgres
-        sudo -u postgres bash -c "psql -c \"CREATE USER pguser WITH PASSWORD 'test1234';\""
-        sudo -u postgres createdb testdb -O pguser
+        # Modify pg_hba.conf to allow trust authentication for devzero
+        echo "local   all             devzero                                trust" | sudo tee -a /etc/postgresql/16/main/pg_hba.conf
+        echo "host    all             devzero        127.0.0.1/32            trust" | sudo tee -a /etc/postgresql/16/main/pg_hba.conf
+        echo "host    all             devzero        ::1/128                 trust" | sudo tee -a /etc/postgresql/16/main/pg_hba.conf
       directory: /home/devzero
       user: root
+launch:
+  steps:
+    - type: command
+      command: |
+        # create the devzero user
+        sudo -u postgres bash -c "psql -c \"CREATE USER devzero;\""
```

Comment:
use the devzero user and you don't need to separately launch postgres service, it'll already happen